tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from django.template import engines
+
+
+@pytest.fixture
+def rusty():
+    return engines["rusty"].from_string
+
+
+@pytest.fixture
+def django_temp():
+    return engines["django"].from_string
+
+
+@pytest.fixture
+def assert_render(rusty, django_temp):
+    def temp(template, context, expected):
+        assert django_temp(template).render(context) == expected
+        assert rusty(template).render(context) == expected
```

Comment:
When `assert_render` fails, is it clear which of these asserts was the cause? I'm a bit cautious that we might try debugging Rust, when actually the Django assert has failed.

Reply:
Output when django fails. So I would say it's clear. 
```
    def temp(template, context, expected):
>       assert django_temp(template).render(context) == expected
E       assert '\\"double qu...le quotes\\\'' == '\\"double qu... q quotes\\\''
E
E         - \"double quotes\" and \'single q quotes\'
E         ?                                --
E         + \"double quotes\" and \'single quotes\'

tests/conftest.py:18: AssertionError
```

Reply:
I'd probably go with `django_template`.